### PR TITLE
Update test to reflect changes done in odo-supervisord-image#27

### PR DESCRIPTION
**Pull request by kadel**
Updates test so they will work with https://github.com/openshift/odo-supervisord-image/pull/27

all information about this change is in  https://github.com/openshift/odo-supervisord-image/pull/27

### CODE REVIEW

**Comment by kadel** [comment]
updated  openshift/odo-supervisord-image#27
triggering all test to verify again

/test all

File: tests/helper/helper_oc.go [hunk 1]
```diff
@@ -110,9 +110,9 @@ func (oc *OcRunner) SourceTest(appTestName string, sourceType string, source str
 }
 
 // ExecListDir returns dir list in specified location of pod
-func (oc *OcRunner) ExecListDir(podName string, projectName string) string {
+func (oc *OcRunner) ExecListDir(podName string, projectName string, dir string) string {
```
Comment (mohammedzee1000) [inline tests/helper/helper_oc.go]: +1

File: tests/helper/helper_oc.go [hunk 2]
```diff
@@ -333,3 +333,19 @@ func (oc *OcRunner) GetEnvFromEntry(componentName string, appName string, projec
 		"-o", "jsonpath='{.spec.template.spec.containers[0].envFrom}'")
 	return strings.TrimSpace(envFromOut)
 }
+
+// GetEnvs returns all env variables in deployment config
+func (oc *OcRunner) GetEnvs(componentName string, appName string, projectName string) map[string]string {
+	var mapOutput = make(map[string]string)
+
+	output := CmdShouldPass(oc.path, "get", "dc", componentName+"-"+appName, "--namespace", projectName,
```
Comment (girishramnani) [inline tests/helper/helper_oc.go]: Can we use api call instead of depending on oc here?

**Pull request merged**